provision/vvv-init.sh

```diff
@@ -5,8 +5,23 @@
 
 echo "Beginning WP Meta Environment provisioning"
 
+# Install wkhtmltopdf for wordcamp-docs plugin
+if [ ! -f /usr/local/bin/wkhtmltopdf ];
+then
+  echo "Installing wkhtmltopdf"
+	apt-get install -y wkhtmltopdf xvfb
+	echo 'xvfb-run --server-args="-screen 0, 1024x768x24" /usr/bin/wkhtmltopdf $*' | sudo tee /usr/bin/wkhtmltopdf.sh > /dev/null
+	chmod a+x /usr/bin/wkhtmltopdf.sh
+	ln -s /usr/bin/wkhtmltopdf.sh /usr/local/bin/wkhtmltopdf
+fi
+
```

Comment:
What's the reason to move this from the wordcamp-specific provisioner?

Reply:
because it's not a site being provisioned, this really should be in a utility